scripts/go_test.sh

```diff
@@ -2,6 +2,17 @@
 
 set -o pipefail
 
-# Run go test and colorize output (PASS - green, FAIL - red).
-# Set "-p 1" to run test sequentially to avoid parallel changes in testing database.
-go test -v -p 1 -coverprofile=coverage.txt -covermode=atomic ./... | sed ''/PASS/s//$(printf "\033[32mPASS\033[0m")/'' | sed ''/FAIL/s//$(printf "\033[31mFAIL\033[0m")/''
+TEST_DIRS=$(go list ./... | grep -v "app/database")
+
+
+function run_tests() {
+  # Run go test and colorize output (PASS - green, FAIL - red).
+  # Set "-p 1" to run test sequentially to avoid parallel changes in testing database.
+  go test -v -p 1 -coverprofile=coverage.txt -covermode=atomic $1 | sed ''/PASS/s//$(printf "\033[32mPASS\033[0m")/'' | sed ''/FAIL/s//$(printf "\033[31mFAIL\033[0m")/''
+}
+
+# Run normal tests
+run_tests "$TEST_DIRS"
+
+# Run destructive schema compatiblity test
+run_tests app/database
```

Comment:
So will this remove testing data and schema? I use testing database after running tests in docker-compose. Testing database initialization is the useful side effect of tests running. Also I am not sure if it's good to send coverage report twice. It's currently broken. I reported it to @mkoura.

Reply:
I didn't think of that. Yes it does.

I think we need to have this test. At least in the CI pipeline.

Reply:
Could you modify this test so it will leave the data unchanged? You can process 1) drop data 2) down pipeline - 3) check - 4) up pipeline - 5) check and finally apply test_data.sql again. It allows you to include this test among the others. So you will send coverage report only once.

Reply:
Thanks, Schema test now runs before unit tests.